spring-integration-core/src/main/java/org/springframework/integration/support/leader/LockRegistryLeaderInitiator.java

```diff
@@ -168,6 +174,17 @@ public LockRegistryLeaderInitiator(LockRegistry locks, Candidate candidate) {
 		this.candidate = candidate;
 	}
 
+	/**
+	 * Sets the {@link ExecutorService}, where is not provided then a default of
```

Comment:
You don't listen to me: must be imperative. Therefore `Set`, not `Seets`

> I was following other setter docs in the class

Then the other one is wrong and must be fixed as well.

Thanks

Reply:
OK I'll update both my doc and the existing one.

I was confused by you saying mine didn't follow convention when then existing didn't either but was already in the source.

Reply:
Yeah... Sorry for confuse.
What I can tell about that code that it was there before obtaining conventions for the JavaDocs.
Or we have just missed it during review 😄 

Good to have you with a perceptive eye to spot and point for that issue 😉 !